README.md

```diff
@@ -1,15 +1,74 @@
-# Knowledge Base Service
+<!-- Improved compatibility of back to top link: See: https://github.com/othneildrew/Best-README-Template/pull/73 -->
+<a name="readme-top"></a>
 
-## Execute using Docker Compose
 
-```
-docker compose up -d --build
-```
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/DewyKB/dewy">
+    <img src="images/logo.png" alt="Logo" width="160" height="160">
+  </a>
+
+<h3 align="center">Dewy - The Knowledgebase for AI</h3>
+
+  <p align="center">
+    Opinionated knowledge extraction and semantic retrieval for Gen AI applications.
+    <br />
+    <a href="https://github.com/DewyKB/dewy"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/DewyKB/dewy/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/github_username/repo_name/issues">Request Feature</a>
+  </p>
+</div>
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+Dewy helps you build AI agents and RAG applications by managing the extraction of knowledge from your documents and implementing semantic search over the extracted content. Load your documents and Dewy takes care of parsing, chunking, summarizing, and indexing for retrieval. Dewy builds on the lessons of putting real Gen AI applications into production so you can focus on getting 💩 done, rather than comparing vector databases and building data extraction infrastructure.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+## Getting Started
```

Comment:
Should refine this to suggest docker first (and move this to a separate `development.md` or `contributing.md`)